tests/testthat/helper-functions.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @param plotList A named list of ggplot2 objects
 animint2HTML <- function(plotList) {
-  res <- animint2dir(plotList, out.dir = "animint-htmltest", 
+  res <- animint2dir(plotList, out.dir = "animint-htmltest",
```

Comment:
please do not commit changes that don't change anything of substance (it just pollutes the diff and makes it harder to understand what is the point of your PR)